charts/redis-ha/values.yaml

```diff
@@ -194,6 +194,9 @@ redis:
   ## It is possible to disable client side certificates authentication when "authClients" is set to "no"
   # authClients: "no"
 
+  # Increase terminationGracePeriodSeconds to 300
+  # terminationGracePeriodSeconds: 300
+
```

Comment:
Update the commend to just reference the Kubernetes doc for the field, uncomment the value and lower the default to something more "normal". Then people who need to increase it can do so with `--set redis.terminationGracePeriodSeconds=300`